backend/src/common/types/repository.type.ts

```diff
@@ -1,5 +1,5 @@
 type Repository<T = unknown> = {
-    find(): Promise<T>;
+    find(userId: number): Promise<T | null>;
```

Comment:
```suggestion
    find(id: number): Promise<T | null>;
```

we would use this method not only in user repo but in file, etc if we want to find something by id